<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlQueryWidget</class>
 <widget class="QWidget" name="SqlQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="sqlSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalHistoryLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="historyLabel">
         <property name="text">
          <string>History</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="historyList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SqlQueryWidget">
      <layout class="QVBoxLayout" name="verticalQueryLayout">
       <item>
        <widget class="QGroupBox" name="groupQuery">
         <property name="title">
          <string>SQL Query</string>
         </property>
         <layout class="QVBoxLayout" name="queryLayout">
          <item>
           <widget class="QTextEdit" name="textSql"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="buttonLayout">
            <item>
             <widget class="QPushButton" name="btnRun">
              <property name="text">
               <string>Run Query</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnExport">
              <property name="text">
               <string>Export</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="querySpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableResults"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textErrors">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Error output...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
